x-pack/test/api_integration/apis/ml/modules/setup_module.ts

```diff
@@ -40,16 +43,56 @@ export default ({ getService }: FtrProviderContext) => {
             jobId: 'pf1_low_request_rate',
             jobState: JOB_STATE.CLOSED,
             datafeedState: DATAFEED_STATE.STOPPED,
+            modelMemoryLimit: '10mb',
           },
           {
             jobId: 'pf1_response_code_rates',
             jobState: JOB_STATE.CLOSED,
             datafeedState: DATAFEED_STATE.STOPPED,
+            modelMemoryLimit: '10mb',
           },
           {
             jobId: 'pf1_url_scanning',
             jobState: JOB_STATE.CLOSED,
             datafeedState: DATAFEED_STATE.STOPPED,
+            modelMemoryLimit: '10mb',
+          },
+        ],
+      },
+    },
+    {
+      testTitleSuffix:
+        'for sample logs dataset with prefix, startDatafeed false and estimateModelMemory true',
+      sourceDataArchive: 'ml/sample_logs',
+      indexPattern: { name: 'kibana_sample_data_logs', timeField: '@timestamp' },
+      module: 'sample_data_weblogs',
+      user: USER.ML_POWERUSER,
+      requestBody: {
+        prefix: 'pf2_',
+        indexPatternName: 'kibana_sample_data_logs',
+        startDatafeed: false,
+        estimateModelMemory: true,
```

Comment:
probably worth to omit this parameter to validate that `estimateModelMemory` is enabled by default